components/image/slider/src/index.js

```diff
@@ -0,0 +1,63 @@
+import React, {PropTypes} from 'react'
+import ReactSlidy from 'react-slidy'
+
+const ImageSlider = (props) => {
+
+  const slides = getSlides(props.images)
+
+  return (
+    <div onClick={props.handleClick} className="sui-ImageSlider">
+      { hasMoreThanOneImage(props.images)
+        ? (<ReactSlidy>{ slides }</ReactSlidy>)
+        : slides
+      }
+    </div>
+  )
+}
+
+/**
+ * Says if there are more than one image in the given list.
+ * Implemented for semantic purposes.
+ * @param {Array} images List given by props.images.
+ * @return {boolean}
+ */
+const hasMoreThanOneImage = (images) => (images && 1 < images.length)
```

Comment:
if you do that you would re define the method on each render.

This way you defined the method only once. Since there is no requirement to re-define the method on render i would say it's fine this way